content/lessons/chapter-4/public-key-4.tsx

```diff
@@ -57,13 +57,18 @@ export default function PublicKey4({ lang }) {
 
   const javascript = {
     program: `
-console.log(compressPublicKey(${prevData?.data && `"${dataObject}"`}))
+console.log(compressPublicKey(uncompressedKey))
 console.log("KILL")`,
     defaultFunction: {
       name: 'compressPublicKey',
       args: ['publicKey'],
     },
-    defaultCode: `${prevData?.data && `const uncompressedKey = "${dataObject}"`}
+    defaultCode: `${
+      prevData?.data &&
+      `const uncompressedKey = ["${dataObject
```

Comment:
Should this be an array?   In the previous code this followed a similar pattern to the python `uncompressed_key` object in line 110.

Reply:
I think the change to array is intentional. Here's some context from the ticket: https://github.com/saving-satoshi/saving-satoshi/issues/971#issuecomment-2161230315

Reply:
I changed it to use an array instead of a string with a comma because it required the user to do some rather unusual transformation I found not relevant to this lesson

Reply:
see this code
```
  const regex = /([a-f0-9]+),([a-f0-9]+)/;
  const cleanedPublicKey = publicKey.replace(/\\s/g, '');
  const match = cleanedPublicKey.match(regex);
  ```